packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -64,6 +64,11 @@ const propTypes = {
    */
   isRowHeader: PropTypes.bool,
 
+  /**
+   * Boolean indicating that the cell is has been highlighted.
+   */
+  isHighlighted: PropTypes.bool,
```

Comment:
Yes, I named it this way because technically a Cell component should not have knowledge of the row.  It just needs to know that it is highlighted, which would occur by hover or row selection.